erpnext_com/www/docs/user/manual/en/setting-up/print/print-settings.md

```diff
@@ -1,13 +1,10 @@
 # Print Settings
 
-In Print Settings you can set your default printing preferences like Paper Size, default text size, whether you want to output as PDF or HTML etc.
-
-To edit print settings, go to:
```

Comment:
line 3, 2nd sentence should be 1. How to edit print settings. refer any doc from CRM, selling, buying, or stock to see new structure

Reply:
I have followed this https://erpnext.com/docs/user/manual/en/buying/buying-settings. 

this seems to be the applicable structure. Also, I have now added numbering for the features.